packages/framework/aqueduct/package.json

```diff
@@ -16,6 +16,7 @@
 	"types": "dist/index.d.ts",
 	"files": [
 		"dist/**/*",
+		"!dist/test/**",
 		"lib/**/*"
```

Comment:
Do we need to exclude `lib/test` for this (and others) too?

Reply:
Don't ask me why 😄, but the test files do not make it into lib. Need to get more familiar with what/how generates lib. But we don't seem to need that piece.

Reply:
Test files aren't generated in both ESNext and es6, like the src is. Lib contains the esnext content.